src/clipboard.ts

```diff
@@ -15,13 +15,49 @@
  */
 import { schema } from '@manuscripts/transform'
 import { DOMParser } from 'prosemirror-model'
+
+export const getJatsListType = (cssStyle: string | null) => {
```

Comment:
maybe that function should be in transform?

Reply:
done, moved to transform [PR](https://github.com/Atypon-OpenSource/manuscripts-transform/pull/192)